src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int firstSide;
+    private int secondSide;
+
+    public Rectangle(String color, int firstSide, int secondSide) {
+        this.color = color;
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+    }
+
+    @Override
+    public double getArea() {
+        return firstSide * secondSide;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: rectangle, color: " + getColor() + ", area: " + getArea()
```

Comment:
Avoid using concatenation in `System.out.println` calls for better performance and readability. Consider using `printf` or `String.format`.